<launch>

  
  <!-- Define args -->
  
  <arg name="odometry_topic"                    default="/gt/odom"/>
  <arg name="true_mbes_topic"                   default="/gt/mbes_pings"/>
  <arg name="simulated_mbes_topic"              default="/sim/mbes"/>
  <arg name="output"                            default="screen"/>



  <node type="auv_2_listener.py" pkg="auv_2_ros" name="auv_2_listener" output="$(arg output)">

    <param name="odometry_topic"                          value="$(arg odometry_topic)" />
    <param name="true_mbes_topic"                         value="$(arg true_mbes_topic)" />
    <param name="simulated_mbes_topic"                    value="$(arg simulated_mbes_topic)" />

  </node>
  
</launch>